src/Umbraco.Web.UI.Client/src/common/services/contenteditinghelper.service.js

```diff
@@ -210,7 +199,7 @@ function contentEditingHelper(fileManager, $q, $location, $routeParams, notifica
 
             //This is the ideal button order but depends on circumstance, we'll use this array to create the button list
             // Publish, SendToPublish, Save
-            var buttonOrder = ["U", "H", "A"];
+            var buttonOrder = ["U", "H"];
```

Comment:
@madsrasmussen maybe also remove the last part of the comment with "Save" :)

Reply:
You have eagle eyes my friend. It is now fixed 😃